front/app/components/admin/participation/index.tsx

```diff
@@ -21,26 +23,45 @@ const ProjectParticipation = () => {
   const { projectId } = useParams() as { projectId: string };
   const { pathname } = useLocation();
   const { formatMessage } = useIntl();
+  const { data: appConfiguration } = useAppConfiguration();
 
   const basePath = `/admin/projects/${projectId}/audience`;
 
-  const tabs: ITab[] = [
-    {
-      name: 'participants',
-      label: formatMessage(messages.participantsTab),
-      url: `/admin/projects/${projectId}/audience` as RouteType,
-    },
-    {
-      name: 'demographics',
-      label: formatMessage(messages.demographicsTab),
-      url: `/admin/projects/${projectId}/audience/demographics` as RouteType,
-    },
-    {
-      name: 'traffic',
-      label: formatMessage(messages.trafficTab),
-      url: `/admin/projects/${projectId}/audience/traffic` as RouteType,
-    },
-  ];
+  const privateAttributesInExport =
+    appConfiguration?.data.attributes.settings.core
+      .private_attributes_in_export;
+
+  // Build tabs based on whether participants are visible
+  const tabs: ITab[] = privateAttributesInExport
+    ? [
+        {
+          name: 'participants',
+          label: formatMessage(messages.participantsTab),
+          url: `/admin/projects/${projectId}/audience` as RouteType,
+        },
+        {
+          name: 'demographics',
+          label: formatMessage(messages.demographicsTab),
+          url: `/admin/projects/${projectId}/audience/demographics` as RouteType,
+        },
+        {
+          name: 'traffic',
+          label: formatMessage(messages.trafficTab),
+          url: `/admin/projects/${projectId}/audience/traffic` as RouteType,
+        },
+      ]
+    : [
+        {
+          name: 'demographics',
+          label: formatMessage(messages.demographicsTab),
+          url: `/admin/projects/${projectId}/audience` as RouteType,
+        },
+        {
+          name: 'traffic',
+          label: formatMessage(messages.trafficTab),
+          url: `/admin/projects/${projectId}/audience/traffic` as RouteType,
+        },
+      ];
```

Comment:
Think you can get rid of the code duplication here...

```suggestion
  const privateAttributesInExport =
    appConfiguration?.data.attributes.settings.core
      .private_attributes_in_export !== false;

  // Participants tab only visible if private attributes is enabled
  const participantsTab: ITab[] = privateAttributesInExport
    ? [
        {
          name: 'participants',
          label: formatMessage(messages.participantsTab),
          url: `/admin/projects/${projectId}/audience` as RouteType,
        },
      ]
    : [];

  const tabs: ITab[] = [
    ...participantsTab,
    {
      name: 'demographics',
      label: formatMessage(messages.demographicsTab),
      url: `/admin/projects/${projectId}/audience/demographics` as RouteType,
    },
    {
      name: 'traffic',
      label: formatMessage(messages.trafficTab),
      url: `/admin/projects/${projectId}/audience/traffic` as RouteType,
    },
  ];
```

Reply:
I didn’t do it this way because I made the demographics tab the new root of the `/audience` path when the FF is disabled.

The main reason is to avoid having to manage the tab-handling logic in two different places. If we were to keep the demographics tab at `/audience/demographics`, then the path leading to the audience page itself would also need to become conditional based on the FF.

Would that approach be preferable ?